link_bot_gazebo/IntPoint2D rowcol
---
geometry_msgs/Vector3 xyz
